### PR TITLE
Began on curve-snake game

### PR DESCRIPTION
**Pull request by angloth**
HMmm:

* 1
* 2
* 3

### CODE REVIEW

**Review by Bence657 — CHANGES_REQUESTED** [review]
* Fel
* Fel
* ojojoj

File: user_programs/imperialism.cmm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// left arrow = 128, down, up, right
+// w = 119, a = 97, s = 115, d = 100
+
+
+int get_input(){
+
+}
+
+int imperialism(){
```
Comment (Bence657) [inline user_programs/imperialism.cmm:9]: Broooo this code is really bad

**Review by Bence657 — APPROVED** [review]
Ok!

**Comment by angloth** [comment]
Test pull request, will be removed and closed

**Pull request closed without merging**